qcontexttabwidget.cpp

```diff
@@ -0,0 +1,63 @@
+// Based on qmditabwidget by Diego Iastrubni
+// https://code.google.com/p/qtedit4/source/browse/tools/qmdilib/src/qmditabwidget.cpp
+
+#include "qcontexttabwidget.h"
+
+#include <QTabBar>
+#include <QEvent>
+#include <QMouseEvent>
+
+QContextTabWidget::QContextTabWidget(QWidget* parent)
```

Comment:
Please rename the widget in `ContextTabWidget`, to avoid possible confusion with other Qt widgets.